cluster/node-pools/master-default/files.yaml

```diff
@@ -51,3 +51,11 @@ files:
     data: "{{ .Cluster.ConfigItems.service_account_private_key | base64Decode | publicKey | base64 }}"
     permissions: 0400
     encrypted: false
+  - path: /var/www/openid-configuration/discovery.json
+    data: "{{ generateOidcDiscoveryDocument .Cluster.LocalID .Values.hosted_zone | base64 }}"
```

Comment:
nit: function name should be `generateOIDCDiscoveryDocument` to follow the convention we use from go.